apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/internal/DeferredJsonMerger.kt

```diff
@@ -50,12 +50,27 @@ class DeferredJsonMerger {
     // payloadData can be null if there are errors
     if (payloadData != null) {
       val nodeToMergeInto = nodeAtPath(mergedData, payloadPath) as MutableMap<String, Any?>
```

Comment:
What is guaranteeing that the cast to MutableMap is safe?

Reply:
ALright, because it's all coming from `_merged = mutableMapOf<String, Any?>()` above 👍